text/0265-event-vision.md

```diff
@@ -0,0 +1,74 @@
+# Event basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to write down and agree on a few basics.
+
+### What are events?
+
+Events are a type of log which has a required event name and a specific structure which is implied by that event name.
+
+The event name and its implied structure make events much more suitable for observability compared to generic logs.
```

Comment:
It would be great if we had _something_ to backup this claim that events >>> logs for observability.

Reply:
I think the structure referenced on the line above is the reason. Logs (unstructured) are more difficult to parse, index, and aggregate than events (structured).

Reply:
And the required "event name" makes it easy for backends/users to differentiate between different events.